packages/create-pantheon-decoupled-kit/src/templates/next-drupal/pages/api/preview.js

```diff
@@ -44,6 +45,10 @@ const preview = async (req, res) => {
 				message: error.message,
 			});
 		}
+
+		//  Clear access token
+		delete store.token.accessToken;
```

Comment:
I think we'll want to do this before the call to getObjectByPath, because if that fails this line won't get called and the subsequent call may have the cached token.